package.json

```
@@ -4,7 +4,8 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "make": "yarn && cd ./assembly && yarn && cd .. && cd ./websites/docs.temporal.io && yarn",
+    "make": "chmod +x make.sh && ./make.sh",
```

```suggestion
    "make": "./make.sh",
```

i believe if you commit a file with executable permissions, it will have them on clone